go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/aykevl/tinygl
 
-go 1.20
+go 1.22.0
```

Comment:
Is this intentional?

Reply:
Running `go mod tidy` keeps changing to that, even if I delete the `go.sum`.

Reply:
I see why. Created https://github.com/tinygo-org/tinygl-font/pull/4 to address this.

Reply:
Fixed.